db/structure.sql

```diff
@@ -843,7 +843,8 @@ CREATE TABLE public.phases (
     how_to_enter_delta text,
     inserted_at timestamp(0) without time zone NOT NULL,
     updated_at timestamp(0) without time zone NOT NULL,
-    submissions_count integer DEFAULT 0 NOT NULL
+    submissions_count integer DEFAULT 0 NOT NULL,
+    active_submissions_count integer DEFAULT 0 NOT NULL
```

Comment:
this column needs to be removed since it is no longer part of the schema. you can drop it directly from your database, or create and run a migration that drops it and then delete the migration since it won't be committed/merged.
```suggestion
```